.github/workflows/ngc_reusable.yml

```diff
@@ -0,0 +1,66 @@
+name: NGC Reusable workflow
+
+on:
+  workflow_call:
+    secrets:
+      ALIYUN_DOCKER_USERNAME:
+        required: false
+      ALIYUN_DOCKER_PASSWORD:
+        required: false
+      DOCKER_USERNAME:
+        required: false
+      DOCKER_PASSWORD:
+        required: false
+
+jobs:
+  TORCH-NGC:
+    if: github.repository == 'alibaba/BladeDISC' && github.event.ref == 'refs/heads/main'
+    # The type of runner that the job will run on
+    runs-on: [self-hosted, gpu-t4]
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Runs a single command using the runners shell
+    - name: Checkout
+      uses: actions/checkout@v2.4.0
+    - name: Build Dev Docker
+      run: |
+        set -e
+        git submodule sync
+        git submodule update --depth=1 --init --recursive
+        # docker pull bladedisc/bladedisc:latest-devel-pt-ngc
+        # docker build --cache-from bladedisc/bladedisc:latest-devel-pt-ngc -t disc-dev-pt-ngc \
+        docker build -t disc-dev-pt-ngc \
+          --build-arg BASEIMAGE=nvcr.io/nvidia/pytorch:21.11-py3 -f docker/dev/Dockerfile.torch.ngc .
+    - name: Build and Test DISC
+      run: |
+        set -e
+        mkdir -p $HOME/.cache/.torch/
+        mkdir -p $HOME/.cache/.torch/.cache
+        cp $HOME/.cache/remote_cache_token $HOME/.cache/.torch/.cache
+
+        nvidia-docker run --rm -t --user $(id -u) \
+          -v $HOME/.cache/.torch/:$HOME/.torch \
+          -v $HOME/.cache/.torch/.cache:$HOME/.cache \
```

Comment:
Does a new cache dir increase the disc usage of the CI host?  How about only using the remote cache instead of mounting any host cache dir?

Reply:
Yes, it would increase the disk usage, but not too much. 
![image](https://user-images.githubusercontent.com/8392447/160050535-9759bf6e-99fe-43d3-930e-b4a85c397a2b.png)

I think we can do some experiments on disc_cache and remote_cache, with a little sacrifice of disk space. If we finally come to a conclusion that remote_cache is superior to disk_cache then when just deprecate it.

Reply:
`--disc_cache` overwrites remote cache, see https://docs.bazel.build/versions/main/remote-caching.html#disk-cache . But I think the problem behind the sense is that our cache could NOT deal with multiple CUDA version correctly.